torch/_inductor/scheduler.py

```diff
@@ -577,6 +577,8 @@ def get_buf_bytes(buf: Optional[Union[ir.Buffer, ir.TensorBox]]) -> int:
                     users = self.scheduler.name_to_buf[buf.get_name()].users
                     tot = 0
                     for user in users:
+                        if isinstance(user.node, OutputNode):
+                            return 0
```

Comment:
I am not familiar enough with this code to know what this is doing haha (should someone from inductor take a look?)

Reply:
updated the code a bit, and will ask Horace to help take a look at this